tests/product/base_product_case.py

```diff
@@ -402,7 +424,7 @@ def assert_started(self, process_per_host):
         return process_per_host
 
     def escape_for_regex(self, expected):
-        expected = expected.replace('{rpm}', PRESTO_RPM)
+        expected = expected.replace('{rpm}', self.presto_rpm_filename)
```

Comment:
1. There's one case where the basename isn't quite that (but I guess in that case, it's not actually the basename then). I can change it.
2. That escapes too many things and I got a regex parsing error when I used it!
3. yes. I did it like this because I didn't want to change all of the text files to escape the []() characters.